src/components/PopUp.tsx

```diff
@@ -0,0 +1,74 @@
+import React, { useState } from "react";
+import { View, StyleSheet, ViewStyle } from "react-native";
+import type { StyleProp } from "react-native";
+import { PRIMARY_COLOR, TEXT_COLOR } from "../util/colors";
+import { Button } from "./Button";
+import Modal from "react-native-modal";
+
+export interface ModalProps {
+	text: string;
+	accessibilityLabel: string;
+	accessibilityHint?: string;
+	modalAccessibility: string;
+	closeButtonAccessibilityLabel: string;
+	closeButtonText: string;
+	style?: StyleProp<ViewStyle>;
+}
+
+export const PopUp: React.FC<ModalProps> = ({
+	text,
+	accessibilityLabel,
+	accessibilityHint,
+	modalAccessibility,
+	closeButtonText,
+	closeButtonAccessibilityLabel,
+	style,
+	children,
+}) => {
+	const [modalVisible, setModalVisible] = useState(false);
+	return (
+		<View
+			accessibilityLabel={modalAccessibility}
+			accessibilityHint={modalAccessibility}
```

Comment:
```suggestion
```
having this twice will cause the element to repeat its text twice on Android

Reply:
ahhh, I did not know that! So `accessibilityLabel` is sufficient then? What's the difference? @carpenterd777